scalacheck/src/main/scala/pathy/scalacheck/RandomSeg.scala

```diff
@@ -28,9 +28,10 @@ private[scalacheck] object RandomSeg {
   implicit val randomSegArbitrary: Arbitrary[RandomSeg] =
     Arbitrary {
       Gen.nonEmptyListOf(Gen.frequency(
-        100 -> Arbitrary.arbitrary[Char],
+        100 -> Gen.alphaNumChar,
```

Comment:
What corner cases do you have in mind?

What I actually wanted was something like `100 -> Gen.asciiChar`, but there isn't any such thing as far as I could tell, so I went with `alphaNum` as the best available substitute. If you like that better, I could try to come up with a succinct way to express it. I guess I would still want to filter out things like `\n` and `\t`, though, just because they're hard to read in the test output.

I agree giving a name to the list of escaped chars is helpful.